TestON/tests/FUNCintent/FUNCintent.py

```diff
@@ -150,7 +160,7 @@ def CASE10( self, main ):
                                  onpass="Successfully installed ONOS package",
                                  onfail="Failed to install ONOS package" )
 
-        time.sleep( 20 )
+        time.sleep( 60 )
```

Comment:
I think we need to do this for all time.sleep() statements where ever we use in test scripts. Params file will include them with default values assigned (or working values in our setups), that way an end user would be able change them depending on their testbeds (example: someone running ONOS in AWS may have slow connection)